### ISSUE TITLE
Graylog doesn't send "missing" on aggregation

### ISSUE THREAD

**Issue by samupr** [Graylog2/graylog2-server#14324]
## Expected Behavior
When using "Group by" to group based on multiple fields, if the field is missing on some documents, then the document should still be aggregated with some default value such as "Empty value".

## Current Behavior
Currently, the entire document is aggregated to a single row as "(Empty Value)" even when the missing fields are different. I recently upgraded to Graylog 5.0.1 and now notice this behaviour. I'm almost sure that on version 4, the behaviour was as expected.

## Possible Solution
Send "missing" parameter to opensearch.

## Steps to Reproduce (for bugs)
Create aggregation for messages where some messages are missing different fields and then group by those different fields.

![image](https://user-images.githubusercontent.com/52700349/209797405-30f22b91-1c26-49a7-9f59-bc0495f41971.png)

## Your Environment

* Graylog Version: 5.0.1
* Java Version: 17
* Elasticsearch Version: Opensearch 2.4.1
* MongoDB Version: 5.0
* Operating System: Centos 7

**Comment by giveen** [comment]
I think this is also causing an issue with my alerts/notifications


```
org.graylog.events.processor.EventProcessorException: Couldn't create events for: EventDefinitionDto{scope=DEFAULT, id=63938873432e913ea74be33f, title=Palo Alto Threat Alert, description=, priority=2, alert=true, config=AggregationEventProcessorConfig{type=aggregation-v1, query=source_zone:outside AND event_log_name:THREAT AND !(alert_signature:"SCAN: Host Sweep(8002)") AND !(source_ip_whois_organization:"NCATS (C07252631)") AND !(source_ip_whois_organization:"University of Texas at Austin (UTAA)"), queryParameters=[], streams=[63938495432e913ea74bda46], groupBy=[alert_signature, destination_ip, source_ip, rule_name, alert_indicator], series=[AggregationSeries{id=count-source_ip, function=COUNT, field=Optional[source_ip]}], conditions=Optional[AggregationConditions{expression=Optional[GreaterEqual{expr=>=, left=NumberReference{expr=number-ref, ref=count-source_ip}, right=NumberValue{expr=number, value=5.0}}]}], searchWithinMs=300000, executeEveryMs=60000}, fieldSpec={source_ip=EventFieldSpec{dataType=STRING, providers=[Config{type=template-v1, template=${source.source_ip}, requireValues=false}]}, alert_signature=EventFieldSpec{dataType=STRING, providers=[Config{type=template-v1, template=${source.alert_signature}, requireValues=false}]}, destination_ip=EventFieldSpec{dataType=STRING, providers=[Config{type=template-v1, template=${source.destination_ip}, requireValues=false}]}, rule_name=EventFieldSpec{dataType=STRING, providers=[Config{type=template-v1, template=${source.rule_name}, requireValues=false}]}, alert_indicator=EventFieldSpec{dataType=STRING, providers=[Config{type=template-v1, template=${source.alert_indicator}, requireValues=false}]}}, keySpec=[source_ip, alert_signature, destination_ip, rule_name], notificationSettings=EventNotificationSettings{gracePeriodMs=1800000, backlogSize=1}, notifications=[Config{notificationId=63938873432e913ea74be33c, notificationParameters=Optional.empty}], storage=[Config{type=persist-to-streams-v1, streams=[000000000000000000000002]}]}
        at org.graylog.events.processor.EventProcessorEngine.execute(EventProcessorEngine.java:105) ~[graylog.jar:?]
        at org.graylog.events.processor.EventProcessorExecutionJob.execute(EventProcessorExecutionJob.java:115) ~[graylog.jar:?]
        at org.graylog.scheduler.JobExecutionEngine.executeJob(JobExecutionEngine.java:173) ~[graylog.jar:?]
        at org.graylog.scheduler.JobExecutionEngine.lambda$handleTrigger$2(JobExecutionEngine.java:151) ~[graylog.jar:?]
        at com.codahale.metrics.Timer.time(Timer.java:151) ~[graylog.jar:?]
        at org.graylog.scheduler.JobExecutionEngine.handleTrigger(JobExecutionEngine.java:151) ~[graylog.jar:?]
        at org.graylog.scheduler.JobExecutionEngine.lambda$execute$0(JobExecutionEngine.java:120) ~[graylog.jar:?]
        at org.graylog.scheduler.worker.JobWorkerPool.lambda$execute$0(JobWorkerPool.java:121) ~[graylog.jar:?]
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:180) [graylog.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:?]
        at com.codahale.metrics.InstrumentedThreadFactory$InstrumentedRunnable.run(InstrumentedThreadFactory.java:66) [graylog.jar:?]
        at java.lang.Thread.run(Unknown Source) [?:?]
**Caused by: java.lang.IllegalArgumentException: Invalid format: "(Empty Value)**"
```

**Comment by samupr** [comment]
I have upgraded to v5.0.3 and my problem is now fixed.
Possibly resolved by https://github.com/Graylog2/graylog2-server/pull/14446

**Comment by dennisoelkers** [comment]
Thanks for the feedback, @samupr!

Fixed in #14446.

**Comment by giveen** [comment]
> I have upgraded to v5.0.3 and my problem is now fixed. Possibly resolved by #14446 @giveen please confirm if updating resolved your problem.

Yes, it has resolved my issue. My notifications/alerts are now functional.
